eng/liveBuilds.targets

```diff
@@ -221,6 +221,6 @@
             ResolveLibrariesRuntimeFilesFromLocalBuild" />
 
   <PropertyGroup>
-    <BundledRuntimeIdentifierGraphFile>$(RuntimeIdGraphDefinitionFile)</BundledRuntimeIdentifierGraphFile>
+    <BundledRuntimeIdentifierGraphFile>$(LiveRuntimeIdGraphDefinitionFile)</BundledRuntimeIdentifierGraphFile>
```

Comment:
By switching to the live generated runtime.json file, you are depending on the file already being generated which isn't the case until the Microsoft.NETCore.Platforms project is built. We should make sure that any code path that depends on it has a dependency to the Platforms project.